pkg/cli/admin/upgrade/status/workerpool.go

```diff
@@ -242,6 +247,17 @@ func assessNodesStatus(cv *configv1.ClusterVersion, pool mcfgv1.MachineConfigPoo
 	return nodesStatusData, insights
 }
 
+func generateInsight(message string, description string) bool {
+	if message == mco.ReasonOfUnavailabilityMCDWorkInProgress {
+		return false
+	}
+	if message == mco.ReasonOfUnavailabilityNodeNotReady || message == mco.ReasonOfUnavailabilityNodeDiskPressure ||
+		message == mco.ReasonOfUnavailabilityNodeNetworkUnavailable {
+		return description != ""
+	}
```

Comment:
For these cases we have a time when the issue started, let's talk about the suitable threshold to reduce noise: what could be a good duration to start reporting the insight? How about five minutes for `NotReady` and a minute for the other two?

Below these thresholds, we would not generate an insight. We would still display the node as unavaialable and put the messages to its line in the table, but no insight unless the codition persists.